engines/private/module.mk

```diff
@@ -0,0 +1,25 @@
+MODULE := engines/private
+ 
+MODULE_OBJS := \
+	metaengine.o \
+    code.o \
+	cursors.o \
+	symbol.o \
+	funcs.o \
+    grammar.tab.o \
```

Comment:
Indentation is wrong here. Same for the line `code.o`

Reply:
Done.